reproduction-in-domestic-animals

```diff
@@ -0,0 +1,176 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" et-al-min="19" et-al-use-first="19" demote-non-dropping-particle="sort-only" default-locale="en-GB" xmlns="http://purl.org/net/xbiblio/csl">
```

Comment:
`et-al-min="19" et-al-use-first="19"`

et-al-use-first always must be a lower number than et-al-min (because it doesn't make sense to use et-al abbreviation for an item with 19 names and still show all 19 names)